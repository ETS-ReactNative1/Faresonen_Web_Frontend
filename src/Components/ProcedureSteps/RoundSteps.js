import React from "react";
import PropTypes from "prop-types";
import { Icon, Step } from "semantic-ui-react";

const RoundSteps = ({ league, round }) => (
  <Step.Group className="no-print">
    <Step completed>
      <Icon name="checkmark" />
      <Step.Content>
        <Step.Title>Liga valgt</Step.Title>
        <Step.Description>Du har valgt {league}</Step.Description>
      </Step.Content>
    </Step>

    <Step completed>
      <Icon name="checkmark" />
      <Step.Content>
        <Step.Title>Runde valgt</Step.Title>
        <Step.Description>Du har valgt runde {round}</Step.Description>
      </Step.Content>
    </Step>

    <Step>
      <Icon name="print" />
      <Step.Content>
        <Step.Title>Print</Step.Title>
        <Step.Description>Ctrl + P for Ã¥ printe lefsen</Step.Description>
      </Step.Content>
    </Step>

    <Step>
      <Icon name="cocktail" />
      <Step.Content>
        <Step.Title>Bra jobbet</Step.Title>
        <Step.Description>Ta deg en velfortjent pause!</Step.Description>
      </Step.Content>
    </Step>
  </Step.Group>
);

RoundSteps.propTypes = {
  round: PropTypes.number.isRequired,
  league: PropTypes.string.isRequired
};

export default RoundSteps;
