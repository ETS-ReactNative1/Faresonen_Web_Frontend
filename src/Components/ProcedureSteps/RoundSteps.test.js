import React from "react";
import renderer from "react-test-renderer";
import RoundSteps from "./RoundSteps";

describe("<RoundSteps />", () => {
  it("Should render properly", () => {
    const tree = renderer.create(<RoundSteps league="Eliteserien" round={1} />);
    expect(tree).toMatchSnapshot();
  });
});
