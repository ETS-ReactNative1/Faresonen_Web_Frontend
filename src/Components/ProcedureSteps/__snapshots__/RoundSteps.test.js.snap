// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RoundSteps /> Should render properly 1`] = `
<div
  className="ui steps no-print"
>
  <div
    className="completed step"
    onClick={[Function]}
  >
    <i
      aria-hidden="true"
      className="checkmark icon"
      onClick={[Function]}
    />
    <div
      className="content"
    >
      <div
        className="title"
      >
        Liga valgt
      </div>
      <div
        className="description"
      >
        Du har valgt
         
        Eliteserien
      </div>
    </div>
  </div>
  <div
    className="completed step"
    onClick={[Function]}
  >
    <i
      aria-hidden="true"
      className="checkmark icon"
      onClick={[Function]}
    />
    <div
      className="content"
    >
      <div
        className="title"
      >
        Runde valgt
      </div>
      <div
        className="description"
      >
        Du har valgt runde
         
        1
      </div>
    </div>
  </div>
  <div
    className="step"
    onClick={[Function]}
  >
    <i
      aria-hidden="true"
      className="print icon"
      onClick={[Function]}
    />
    <div
      className="content"
    >
      <div
        className="title"
      >
        Print
      </div>
      <div
        className="description"
      >
        Ctrl + P for Ã¥ printe lefsen
      </div>
    </div>
  </div>
  <div
    className="step"
    onClick={[Function]}
  >
    <i
      aria-hidden="true"
      className="cocktail icon"
      onClick={[Function]}
    />
    <div
      className="content"
    >
      <div
        className="title"
      >
        Bra jobbet
      </div>
      <div
        className="description"
      >
        Ta deg en velfortjent pause!
      </div>
    </div>
  </div>
</div>
`;
